ginisdk/src/main/java/net/gini/android/models/Document.java

```diff
@@ -29,12 +29,18 @@
         /**
          * Gini has processed the document, but there was an error during processing.
          */
-        ERROR
+        ERROR,
+        /**
+         * Processing state cannot be identified.
+         */
+        UNKNOWN
     }
 
     public enum SourceClassification {
         SCANNED,
-        NATIVE
+        NATIVE,
+        TEXT,
+        UNKNOWN
     }
```

Comment:
`SANDWICH` case should be also added, right? [Doc](http://developer.gini.net/gini-api/html/documents.html#id5)